common/elasticsearch/elasticsearch.go

```diff
@@ -109,6 +111,11 @@ func CreateElasticSearchService(uri *url.URL) (*ElasticSearchService, error) {
 		return nil, fmt.Errorf("Failed to parser url's query string: %s", err)
 	}
 
+	esSvc.Cluster = ESCluster
+	if len(opts["cluster"]) > 0 {
```

Comment:
I think we should call it `cluster_name` and not just cluster.. it'll be easier to understand imo